.circleci/config.yml

```diff
@@ -124,6 +124,16 @@ jobs:
       - store_artifacts:
           path: fastlane/test_output/report.html
           destination: test_report.html
+  apitest:
```

Comment:
should we specify that this is objc-only? i.e.: `objc_api_test`?

Reply:
Since this works, I'd rather keep as is, and optimize later.